projects/components/src/panel/panel.component.scss

```diff
@@ -21,6 +21,7 @@
   .body {
     flex: 1;
     overflow: auto;
+    height: 100%;
```

Comment:
Any side effects due to this?

Reply:
No side effects that I see, but setting a parent height was required to make some scrolling flexbox stuff work correctly deeper down in the DOM.